src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -1029,5 +1029,23 @@ def validate_key_exists(keyname)
       @logger.debug("Using key-pair: `#{keypair.name}' (#{keypair.fingerprint})")
     end
 
+    def relative_and_validate_disk_size(flavor, resource_pool)
+      if resource_pool['root_disk'].nil?
+        if flavor.disk == 0
+          cloud_error("Flavor `#{resource_pool['instance_type']}' should have at least a 1GB boot disk")
```

Comment:
Fair enough, let's use "VM cloud properties" then. Not sure if operators be able to map this to either `resource_pool.cloud_properties` or `vm_type.cloud_properties`, though :)